src/index.ts

```diff
@@ -21,16 +21,39 @@ program
   .version(pkg.version, "-v, --version")
   .option("--summary", "Prints a summary report")
   .option("-i, --interactive", "Runs in interactive mode.")
+  .option(
+    "--environment [environment]",
+    "Which dependencies to check. (production, development, all)",
```

Comment:
I'm wondering if we should just simplify this to either `production` or `both`. Is there a use case for checking only development dependencies? The reason to do only prod is because a lot of licenses has legal language around distribution and only the prod deps should be getting published to npm or packaged (for example in a container).

Reply:
Sounds reasonable to me, I only added development in for completeness of API compatibility with license-checker.